### PR TITLE
Update dependencies; webpack  2.2.1

### PR DESCRIPTION
**Pull request by nathanstitt**
Tested and believe it's ready for merge now

### CODE REVIEW

File: configs/webpack/base.coffee [hunk 1]
```diff
@@ -121,15 +103,22 @@ makeBuildPlugins = (projectConfig) ->
...
+    new ExtractTextPlugin(
+      styleFilename
+    )
+  ]
+
 
+ path = require 'path'
 
-makePathsBase = (projectConfig) ->
+ makePathsBase = (projectConfig) ->
```
Comment (pandafulmanda) [inline configs/webpack/base.coffee]: I can just make the update

Comment (nathanstitt) [inline configs/webpack/base.coffee]: yah, that's not intentional :)  go for it if you're in there

Comment (pandafulmanda) [inline configs/webpack/base.coffee]: also, why `require 'path'` twice?

**Comment by nathanstitt** [comment]
`mangler was too mangly` gets my vote for best commit message 😁

**Pull request merged**